kong/plugins/log-serializers/basic.lua

```diff
@@ -16,6 +16,7 @@ function _M.serialize(ngx)
   return {
     request = {
       uri = ngx.var.request_uri,
+      new_uri = ngx.var.uri,
```

Comment:
This definitely warrants a more careful approach or at least some discussion. We must make it obvious what the difference between those two values is. Two options:

1. use the same name as the Nginx variables (breaking change), and document them as being such.
2. keep `uri`, but rename `new_uri` to `upstream_uri`. Since we diverge from the Nginx variables, we would need to make them look similar, and would have to include the query string as @p0pr0ck5 suggested. However, there will still exist cases where `uri` will be normalized (path normalization, percent-decoded...), and `upstream_uri` will not. There will also be cases (most of the time probably), where both `uri` and `upstream_uri` will be identical, which could be confusing and/or wasteful.

I tend to lean towards 1., and open this against `next`, as a documented breaking change.

This will also need tests btw :)